test/functional/feature_rbf.py

```diff
@@ -127,6 +133,20 @@ def make_utxo(self, node, amount, confirmed=True, scriptPubKey=DUMMY_P2WPKH_SCRI
 
         return COutPoint(int(txid, 16), 1)
 
+    def create_multiple_input_self_transfer(self, input_utxos, fee_rate, sequence=BIP125_SEQUENCE_NUMBER):
+        """Given two input utxos, create one transaction that spends both of them"""
+        [tx, staging_tx] = list(map(lambda utxo: self.wallet.create_self_transfer(
```

Comment:
As `input_utxos` is a pair instead of arbitrary length, does a function name such as `create_double_input_self_transfer` or `create_pair_input_self_transfer` indicate the intention better?

Reply:
I think this is a good suggestion. I'll make this change as suggested